docs/BUILD.bazel

```diff
@@ -1,50 +1,48 @@
 load("@io_bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
-load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 
 package(default_visibility = ["//visibility:private"])
 
-_RULES = [
+# Targets to document.
+# Must have a coressponding `bzl_library` target in `//rules:BUILD.bazel`.
+_DOC_SRCS = [
     "app_clip",
     "app",
-    "apple_patched",
     "extension",
+    "features",
+    "force_load_direct_deps",
```

Comment:
A number of these of these are internal ones which I don't think we really want to provide as an API contract or have starlark docs about what they do but if other people really want to have this I'm don't think it will change the lack of an API beyond `apple_framework` to the project

Reply:
Features seems useful and imo more of the public API than some of the other things we've said are. 

Do you have a specific list you'd want me to remove?